api-docs/api-reference/methods/roles/get-list-effective-roles-assigned-to-a-user-v2.0.rst

```diff
@@ -0,0 +1,400 @@
+user to which the role was directly assigned, whereas the ``sourceId`` for a
+``USERGROUP`` type would be the ID for the user group.
+
+Assignment type
+^^^^^^^^^^^^^^^
+Role assignments are classified in to three ways based on how the tenants for
+which those assignments apply are determined.
+
+.. list-table:: Source type
+   :header-rows: 1
+   :widths: 20 60
+
+   * - Type
+     - Description
+   * - DOMAIN
+     - Assignment applied to all tenants within the user's domain.
+   * - TENANT
+     - Assignment only applies to tenants to which the role was explicitly
+       assigned.
+   * - RCN
+     - Applies to all tenants across all domains within the RCN for which the
+       RCN role applies.
+
+--------
+Examples
+--------
+**Generic Example Response**
+
+This example is contrived to show the different ``sourceTypes`` in a single
+example, and is not meant to represent an real world example.
+
+JSON::
+
+ {
+    "RAX-AUTH:roleAssignments": {
+        "tenantAssignments": [
+            {
+                "onRole": "1234",
+                "onRoleName": "roleName",
+                 "forTenants": [
+                     "t1",
+                     "t2"
+                 ],
+                "sources": [
+                  {
+                       "sourceType": "USER",
+                       "sourceId": "userId",
+                       "assignmentType": "DOMAIN"
+                       "forTenants": [
+                          "t1",
+                          "t2"
+                       ]
+                  },
+                  {
+                       "sourceType": "USERGROUP",
+                       "sourceId": "UserGroupAId",
+                       "assignmentType": "DOMAIN",
+                       "forTenants": [
+                          "t1",
+                          "t2"
+                       ]
+                  },
+                  {
+                       "sourceType": "USERGROUP",
+                       "sourceId": "UserGroupBId",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t1",
+                          "t2"
+                       ]
+                  },
+                  {
+                       "sourceType": "USERGROUP",
+                       "sourceId": "UserGroupCId",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t1"
+                       ]
+                  },
+                  {
+                       "sourceType": "SYSTEM",
+                       "sourceId": "IDENTITY",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t2"
+                       ]
+                  }
+                ]
+            }
+        ]
+    }
+ }
+
+**Across Domains Assignment Example Response**
+
+Assume:
+
+- d1t1 and d1t2 are tenants within the same domain (Domain 1).
```

Comment:
for all three assumptions, put the tenants ``d1t1``, ``d1t2``, and ``d2t1`` between double backticks